src/components/Grid.tsx

```diff
@@ -5,7 +5,7 @@ import { Option } from "react-dropdown";
 const Spicetify = window.Spicetify;
 
 import { CardItem, CardType, Config, SchemeIni, Snippet, TabItemConfig } from "../types/marketplace-types";
-import { getLocalStorageDataFromKey, generateSchemesOptions, injectColourScheme, generateKey } from "../logic/Utils";
```

Comment:
What's it doing here that's different from what the `generateKey` function does? Could we just update `generateKey`?

Reply:
`generateKey` is depended on by localStorage management. I dare not change that since that would mean breaking theme installation for everyone and removing existing config.

Reply:
Ah fair